lib/matplotlib/animation.py

```diff
@@ -383,8 +383,7 @@ def grab_frame(self, **savefig_kwargs):
                              dpi=self.dpi, **savefig_kwargs)
         except (RuntimeError, IOError) as e:
             out, err = self._proc.communicate()
-            _log.info('MovieWriter -- Error '
-                           'running proc:\n%s\n%s' % (out, err))
+            _log.info('MovieWriter -- Error running proc:\n%s\n%s', out, err)
```

Comment:
This change does not seem to belong in this pr?

Reply:
forgot to undo that; reverted that part.